source/index.txt

```diff
@@ -2,8 +2,12 @@
 MongoDB Node Driver
 ===================
 
+.. facet::
+   :name: genre
+   :values: reference
+
 .. meta:: 
-   :keywords: node.js
+   :keywords: overview, node.js, ORM
```

Comment:
Ok, if ORM might be a popular search term, then I think "ODM" (or the expansions of both terms) should also be included since it's not used in a heading or frequently in the text.

Reply:
Sounds good, I'll add the two expanded terms